### PR TITLE
ccpp: configure package repositories for correct OS

### PR DESCRIPTION
**Pull request by jfilak**
If we run ABRT in a container or if we catch a crash in a chroot, we
need to look for debuginfo packages relevant to the source of the crash.
Hence we need to tell package managers which version of OS we are
interested in.

Signed-off-by: Jakub Filak <jfilak@redhat.com>

### CODE REVIEW

File: src/plugins/abrt-action-install-debuginfo.in [hunk 1]
```diff
@@ -89,6 +90,7 @@ if __name__ == "__main__":
             "Usage: %s [-vy] [--ids=BUILD_IDS_FILE] [--pkgmgr=(yum|dnf)]\n"
             "       [--tmpdir=TMPDIR] [--cache=CACHEDIR[:DEBUGINFODIR1:DEBUGINFODIR2...]] [--size_mb=SIZE]\n"
             "       [-e, --exact=PATH[:PATH]...]\n"
+            "       [--releasever=RELEASEVER]\n"
```
Comment (mhabrnal) [inline src/plugins/abrt-action-install-debuginfo.in]: Please, add the new parameter into man page as well.

Comment (jfilak) [inline src/plugins/abrt-action-install-debuginfo.in]: Done. Thanks!

File: doc/abrt-action-install-debuginfo.txt [hunk 2]
```diff
@@ -38,6 +38,9 @@ OPTIONS
 -e,--exact::
    Download only specified files
 
+-R,--releasever::
+   Set specific version of Operation System
```
Comment (mhabrnal) [inline doc/abrt-action-install-debuginfo.txt]: Didn't you mean Operating System?

Comment (jfilak) [inline doc/abrt-action-install-debuginfo.txt]: Good catch! Yes.

**Pull request merged**